lib/pf/role.pm

```diff
@@ -678,6 +659,10 @@ sub shouldAutoRegister {
             return $role;
         }
     }
+    if ($args->{'profile'}->{'_autoregister'} eq "enabled") { 
```

Comment:
I would rather be able to just write 
if ($args->{'profile'}->{'_autoregister'}) { do stuff; } 
but can't right now because it's either set to 'disabled' or 'enabled' which are both true.